docs/windows.md

```diff
@@ -3,45 +3,44 @@
 ## Development Environment
 
 To setup a development environment in Windows:
-1) Install Git for Windows from https://gitforwindows.org
-1) Go to http://www.msys2.org/ and download the x86_64 installer
+1) Install [Git for Windows](https://gitforwindows.org)
+1) Go to [MSYS2 website](http://www.msys2.org) and download the x86_64 installer
 1) Follow the instructions on the page for setting up the basic environment
 1) Run `C:\msys64\mingw64.exe` - a terminal window should pop up
 
 ```bash
 $ pacman -Suy
 $ pacman -S mingw-w64-x86_64-gcc \
     mingw-w64-x86_64-gtk3 \
-    mingw-w64-x86_64-pkg-config \
+    mingw-w64-x86_64-pkgconf \
     mingw-w64-x86_64-cairo \
     mingw-w64-x86_64-gobject-introspection \
     mingw-w64-x86_64-python \
     mingw-w64-x86_64-python-pip \
     mingw-w64-x86_64-python-setuptools \
-    mingw-w64-x86_64-python-wheel
+    mingw-w64-x86_64-python3.9
```

Comment:
Instead of installing by hand, I installed plain git (`pacman -S git`) and ran `packages/windows/msys2-install.sh` after cloning the repo. That worked.
I did not install Git for Windows yet.

Reply:
Yes, that works, but now try to run pre-commit in your dev environment. I agree for the basic use case of testing and releasing that msys git works fine.

Reply:
Indeed, that's what I've been using it for.